ui/app/styles/components/form-field.scss

```diff
@@ -0,0 +1,3 @@
+.has-error-border {
```

Comment:
I think we have a forms.scss this might be better suited to go in, rather than a new page.

Reply:
Smart! WIll do.